web/client/api/__tests__/CSW-test.js

```diff
@@ -281,4 +283,111 @@ describe("getLayerReferenceFromDc", () => {
         expect(layerRef.type).toBe('arcgis');
         expect(layerRef.url).toBe('http://esri_url');
     });
+
+});
+
+describe("getLayerReferenceFromDc", () => {
+    const threeDLayerRecord = {
+        "boundingBox": {
+            "extent": [
+                11.244154369601063,
+                43.75907090685874,
+                11.265929832205956,
+                43.78084636946362
+            ],
+            "crs": "EPSG:4326"
+        },
+        "dc": {
+            "references": [],
+            "identifier": "test:20230829_test_3dtile_01",
+            "date": "2023-09-22",
+            "title": "Metadato di test per 3D-tile",
+            "abstract": "Breve descrizione della risorsa",
+            "description": "Breve descrizione della risorsa",
+            "type": "dataset",
+            "subject": [
+                "Zone a rischio naturale",
+                "health"
+            ],
+            "format": "3D Tiles",
+            "contributor": "Nome dell'ufficio responsabile del dato",
+            "rights": [
+                "otherRestrictions",
+                "otherRestrictions"
+            ],
+            "language": "ita",
+            "source": "Descrizione della provenienza e del processo di produzione del dato (storia, ciclo di vita, rilevazione, acquisizione, forma attuale, qualità richiesta per garantirne l'interoperabilità)",
+            "temporal": "start=2009-01-01; end=2013-12-31",
+            "URI": {
+                "TYPE_NAME": "DC_1_1.URI",
+                "protocol": "https://registry.geodati.gov.it/metadata-codelist/ProtocolValue/www-download",
+                "description": "access point",
+                "value": "https://3d-layers.s3.eu-central-1.amazonaws.com/3dtiles/centro_storico_di_firenze_-_brass_city_model/tileset.json"
+            }
+        }
+    };
+    let mockAxios;
+    beforeEach((done) => {
+        mockAxios = new MockAdapter(axios);
+        setTimeout(done);
+    });
+    afterEach((done) => {
+        mockAxios.restore();
+        setTimeout(done);
+    });
+    it('test getBboxFor3DLayersToRecords function that handles getting bbox for 3D tile layers from ', (done) => {
```

Comment:
improve this title, it's too generic, which case are you testing exactly?
4 records the first is 3d_tile or what else?